pkg/apis/componentconfig/types.go

```diff
@@ -797,12 +797,16 @@ type KubeControllerManagerConfiguration struct {
 	EnableContentionProfiling bool
 	// clusterName is the instance prefix for the cluster.
 	ClusterName string
-	// clusterCIDR is CIDR Range for Pods in cluster.
+	// (will be) DEPRECATED: // clusterCIDR is CIDR Range for Pods in cluster.
 	ClusterCIDR string
+	// clusterCIDRs is a CIDR Range list for Pods in cluster.
+	ClusterCIDRs []string
 	// serviceCIDR is CIDR Range for Services in cluster.
 	ServiceCIDR string
 	// NodeCIDRMaskSize is the mask size for node cidr in cluster.
```

Comment:
yes, I'm not sure on the timing of when this should be marked as depicated, so adding the (will be) prevents the depricated message.